docs/api/animation/PropertyMixer.html

```diff
@@ -11,7 +11,7 @@
     <h1>[name]</h1>
 
     <div class="desc">
-    Buffered scene graph property that allows weighted accumulation.
+    Buffered scene graph property that allows weighted accumulation; used internally.
```

Comment:
Throughout the rest of the documentation the approach is that if it's accessible to the user, it should be documented, with a note added if it's intended only for internal consumption. I think the same is OK here - undocumented functions will only lead to confusion.